packages/web/components/Patron/PatronTile.tsx

```diff
@@ -90,11 +102,7 @@ export const PatronTile: React.FC<Props> = ({ index, patron }) => {
               <Wrap w="100%" justify="center">
                 {patron.pSeedBalance != null && (
                   <WrapItem>
-                    <MetaTag size="md">
-                      {`pSEED: ${Math.floor(
-                        Number(utils.formatEther(patron.pSeedBalance)),
-                      )}`}
-                    </MetaTag>
+                    <MetaTag size="md">{displayBalance}</MetaTag>
```

Comment:
```suggestion
                    <MetaTag size="md" title={pSeedBalance}>{displayBalance}</MetaTag>
```

Reply:
I agree that this would be useful, and I tried a few different ways of displaying it, from a popover to clicking it to toggle between the two units, to the simple title attribute. Unfortunately they all failed because of the parent `LinkBox`/`LinkOverlay`.